string[] model_names
geometry_msgs/Pose[] model_poses
